pkgs/top-level/all-packages.nix

```diff
@@ -10507,20 +10507,20 @@ with pkgs;
 
   any-nix-shell = callPackage ../shells/any-nix-shell { };
 
-  bash = lowPrio (callPackage ../shells/bash/4.4.nix {
+  bash_4 = lowPrio (callPackage ../shells/bash/4.4.nix {
     binutils = stdenv.cc.bintools;
   });
-  bash_5 = lowPrio (callPackage ../shells/bash/5.1.nix {
+  bash = lowPrio (callPackage ../shells/bash/5.1.nix {
     binutils = stdenv.cc.bintools;
   });
-  bashInteractive_5 = lowPrio (callPackage ../shells/bash/5.1.nix {
+  # WARNING: this attribute is used by nix-shell so it shouldn't be removed/renamed
+  bashInteractive = lowPrio (callPackage ../shells/bash/5.1.nix {
```

Comment:
This `lowPrio` should be moved from `bashInteractive` (5) to `bashInteractive_4`.

Reply:
Well spotted, thank you!

Reply:
fixed.